appshell/mac/English.lproj/MainMenu.xib

```diff
@@ -76,6 +76,12 @@
                                     <action selector="performZoom:" target="-1" id="240"/>
                                 </connections>
                             </menuItem>
+                            <menuItem title="Enter Full Screen" keyEquivalent="f" id="241">
```

Comment:
@madanbn Thanks for reviewing this! Yes you are right about this being added to the View menu. But the problem is "View" menu is not present in our resource files as it is dynamically created from the JS layer. May be we should think about how we can reorganize the View menu to comply with Apple guidelines. But for now this should work. Note that this behavior is observed only on El Capitan. I have created a sample app in Xcode on El Capitan and removed the "View" menu from the resource. The OS then then adding "Enter Full Screen" menu to "Window" menu(without any shortcut). So it looks like the OS first looks for "View" menu, if not tries to add "Enter Full Screen " to "Window" menu.

Since this PR was raised against `master`, I have raised another PR #538 targeting `release` branch. @madanbn Could you please merge that PR.